adapters/kargo/params_test.go

```diff
@@ -34,11 +34,16 @@ func TestInvalidParams(t *testing.T) {
 }
 
 var validParams = []string{
+	`{"placementId": ""}`,
+	`{"placementId": "11523"}`,
 	`{"adSlotID": ""}`,
 	`{"adSlotID": "11523"}`,
 }
 
 var invalidParams = []string{
+	`{"placementId": 42}`,
+	`{"placementId": }`,
+	`{"placementID": "32321"}`,
```

Comment:
Just curious, What should be the behaviour if both placementId and adSlotID are defined?